cps/templates/author.html

```diff
@@ -0,0 +1,65 @@
+{% extends "layout.html" %}
+{% block body %}
+<h2>{{title}}</h2>
+
+{% if author is not none %}
+<section class="author-bio">
+  {%if author['image_url'] is not none %}
+  <img src="{{author.image_url}}" alt="{{author.name}}" class="author-photo pull-left">
```

Comment:
Ah, I think it's an issue with the Goodreads library. Shame. https://github.com/sefakilic/goodreads/issues/7